### PR TITLE
Seed update with csv

### LINKED ISSUE #141: Seed File
- Make CSV for users
- Make CSV for orders
- Orders and order_items in same csv

### PR DESCRIPTION
**Pull request by CjMoore**
- adds user csv to load users
- changes create orders to find_by item title
- can re-seed without resetting database
- closes #141

### CODE REVIEW

**Review by meyerhoferc — APPROVED** [review]
Looks great! Ready for second review & merge @cmacaulay @valerietrudell

File: db/csv/users2.csv [hunk 1]
```diff
@@ -0,0 +1 @@
+email,username,password,first_name,last_name,address,roleemail@email.com,user,user,Standard,User,123 Fake St,defaultadmin@email.com,admin,admin,Admin,User,123 Fake St,admin
```
Comment (cmacaulay) [inline db/csv/users2.csv:1]: 👍

File: db/seeds.rb [hunk 2]
```diff
@@ -15,50 +15,51 @@ def import_product_csv
...
-import_product_csv
 
-user = User.create!(username: "user",
-                    email: "email@email.com",
-                    password: "user",
-                    first_name: "Standard",
-                    last_name: "User",
-                    address: "123 Fake St" )
-puts "User Created!"
+def import_user_csv
```
Comment (cmacaulay) [inline db/seeds.rb:19]: 👍

File: db/seeds.rb [hunk 3]
```diff
@@ -15,50 +15,51 @@ def import_product_csv
...
+
+OrderItem.delete_all
+Item.delete_all
+Order.delete_all
+User.delete_all
+Category.delete_all
+
+import_product_csv
+import_user_csv
+create_orders
```
Comment (cmacaulay) [inline db/seeds.rb:65]: Looks good!

**Review by cmacaulay — APPROVED** [review]

**Pull request merged**